### DISCUSSION TITLE
search engine used on website

### DISCUSSION THREAD

**Discussion by xxxpsyduck** [fastapi/fastapi#6658] [Questions] [answered]
Sorry, just a small question. What is the thing behind your search function on [fastapi.tiangolo.com](https://fastapi.tiangolo.com/)? I find it very smart and quick so I'm a bit curious.

**Comment by chbndrhnns** [comment, marked as answer]
It's using a framework called [Mkdocs](https://www.mkdocs.org/)  which includes the search functionality IIRC.

**Comment by ArcLightSlavik** [comment]
Yeah, https://squidfunk.github.io/mkdocs-material/setup/setting-up-site-search/ or https://www.mkdocs.org/user-guide/configuration/#search.
